prospector/commit_processor/feature_extractor.py

```diff
@@ -29,13 +31,17 @@ def extract_features(
 
     changes_relevant_path = extract_changes_relevant_path(commit, advisory_record)
     other_CVE_in_message = extract_other_CVE_in_message(commit, advisory_record)
+    referred_to_by_nvd = extract_referred_to_by_nvd(
+        commit, advisory_record, "http://127.0.0.1:8000"
```

Comment:
We would need to refactor this, the URL of the backend should come from the configuration or command line flags...
However, this would affect the interface of one or more functions, we might address that as a separate PR; what do you think? For sure, we must not forget this :-)